File: January 2023/09 January 2023/C++/Last modified ball.cpp
//{ Driver Code Starts
// Initial Template for C++

#include <bits/stdc++.h>
using namespace std;

// } Driver Code Ends

class Solution {
  public:
    int solve(int N, vector<int> A) {

        for (int i = N - 1; i >= 0; i--)
            if (A[i] != 9)
                return i + 1;
    }
};

//{ Driver Code Starts.

int main() {
    int T;
    cin >> T;
    while (T--) {
        int N;
        cin >> N;
        vector<int> A(N);

        for (int i = 0; i < N; i++) cin >> A[i];

        Solution ob;
        cout << ob.solve(N, A) << "\n";
    }
    return 0;
}
    
// } Driver Code Ends

File: Problems & Solutions/L/Last modified ball/09 January 2023/Python/Last modified ball.py
class Solution():
    def solve(self, N, A):
        c = 1
        for i in range(n-1,-1,-1):
            if A[i] != 9:
                break
        return i+1

#{ 
 # Driver Code Starts
#Initial Template for Python 3

for _ in range(int(input())):
    n = int(input())
    array=[int(i) for i in input().split()]
    obj = Solution()
    print(obj.solve(n, array))
# } Driver Code Ends